src/UserService.Mappers/Models/CreateImageDataMapper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.UserService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.UserService.Mappers.Models
+{
+  public class CreateImageDataMapper : ICreateImageDataMapper
+  {
+    public List<CreateImageData> Map(List<AddImageRequest> request, Guid senderId)
+    {
+      List<CreateImageData> result = new();
+
+      if (request == null)
+      {
+        result = null;
+      }
+      else
+      {
+        foreach (AddImageRequest image in request)
+        {
+          result.Add(new CreateImageData(image.Name, image.Content, image.Extension, senderId));
+        }
+      }
+
+      return result;
+    }
+
+    public List<CreateImageData> Map(string name, string content, string extension, Guid senderId)
```

Comment:
why do wee need second map that in fact the same as first with List<AddImageRequest>? we can use only first mapper and when we want to map only one image - we ca use first with one element in list

Reply:
just not to write smth like
Map(new List<AddImageRequest> {new AddImageRequest
  {
     name = "name",
     content = "asdas",
     extension = ".jpg"
  },
senderId});